devel/nodejs10/Portfile

```diff
@@ -61,6 +67,12 @@ post-patch {
     reinplace "s|python|${configure.python}|" ${worksrcpath}/deps/v8/gypfiles/toolchain.gypi
 }
 
+# use the system libuv instead of the bundled version, as it is fixed for older systems
+if { ${os.platform} eq "darwin" && ${os.major} < 11 } {
+    depends_lib-append    path:lib/libuv.dylib:libuv
+    configure.args-append --shared-libuv
```

Comment:
Why not just use macports libuv on all systems ? Generally I always try and favour using macports deps over builtin versions as far as possible, and if upstream has the option to allow it, why not use it everywhere ?

Reply:
That is what I would do, if @ci42 wants to -- but then we would have to revbump all the current installations, rather than just build on the broken systems.

When i do these "older systems" support PRs, I admit that I try to be an unobtrusive as possible, so as not to raise any significant alarms and avoid as much push-back as possible.

Reply:
Thats understandable. Lets see what @ci42 thinks.

Reply:
When I took over the node port (version 0.x) there was no libuv port. When there was one, it wasn't always up to date, so I decided to use the bundled libuv. Since then I haven't bothered about it anymore. If the libuv port is now always up to date and well maintained, I don't mind using it.